cluster-http/src/test/scala/akka/cluster/http/management/scaladsl/ClusterHttpManagementRoutesSpec.scala

```diff
@@ -343,34 +352,29 @@ class ClusterHttpManagementRoutesSpec
           TestShardedActor.extractEntityId,
           TestShardedActor.extractShardId
         )
-        val initializeEntityActorAsk = shardRegion.ask("hello")(Timeout(3.seconds)).mapTo[String]
-        Await.result(initializeEntityActorAsk, 3.seconds)
+
+        val t = ScalatestTimeout(3.seconds)
+
+        shardRegion.ask("hello")(Timeout(3.seconds)).mapTo[String].futureValue(t)
```

Comment:
the `(t)` can come in implicitly I think. Since it's just timeouts, perhaps we could use 5 seconds of all of those